### ISSUE TITLE
Color coding of categories

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/narr-next#22]
We will have a library of colors.  Each category will have some decoration in a color.  Entries created for that category will have have that same color in their visualisation.

Let's start with, say, 10 categories.

**Comment by shahbhumi53** [comment]
I don't understand about which categories you are talking about!

**Comment by IanMayo** [comment]
Each category is one of the "group" boxes on the right hand side.

**Comment by shahbhumi53** [comment]
so there are total 5 categories right now! and you want the header color of each category different?

**Comment by IanMayo** [comment]
Yes, in 5 unique colours. Then, we "tag" entries of that kind in the listing with some slash of colour, to remind the user which category they came from.

**Comment by shahbhumi53** [comment]
okay so color of privacy and type will be according to the event?

**Comment by IanMayo** [comment]
The color of "something" will be according to the event.  Maybe one of those badges.  Maybe a "lozenge" somewhere else in the entry.

**Comment by IanMayo** [comment]
I've introduced a border around each entry.  We could shade this border with the unique color for that category.

So, in this following diagram, we would shade the heading of **General** with one of our colors.  Then, each entry triggered from a **General** button would have an outline in that shade.

<img width="630" alt="screen shot 2018-03-28 at 21 00 31" src="https://user-images.githubusercontent.com/1108513/38053468-1a78be66-32cc-11e8-8ba8-cfd79a307fc9.png">
